SignalServiceKit/src/Util/AppReadiness.m

```diff
@@ -94,10 +94,12 @@ - (void)runAppReadyBlocks
     OWSAssertIsOnMainThread();
     OWSAssert(self.isAppReady);
 
-    for (AppReadyBlock block in self.appReadyBlocks) {
+    // Make a local copy, then clear this state.
```

Comment:
Is this pertinent to the fix? It's all main thread only right?

A comment as to "why" might be more helpful than "what"

Reply:
Having thought it over, I think I'm going to revert this change.